multiuser/keycloak/che-multiuser-keycloak-user-remover/src/main/java/org/eclipse/che/multiuser/keycloak/server/KeycloakUserRemover.java

```diff
@@ -69,23 +69,22 @@ public KeycloakUserRemover(
       @Nullable @Named("che.keycloak.cascade_user_removal_enabled") boolean userRemovalEnabled,
       @Nullable @Named("che.keycloak.admin_username") String keycloakUser,
       @Nullable @Named("che.keycloak.admin_password") String keycloakPassword,
-      KeycloakSettings keycloakSettings,
-      OIDCInfoProvider oidcInfoProvider,
+      @Nullable @Named(REALM_SETTING) String realm,
```

Comment:
what is this new env var about and how is it documented?

Reply:
That's known property. But if realm is null on the hosted Che, then I  think `remove user` feature will be broken when this feature is enabled...  I see that realm is required for https://github.com/eclipse/che/blob/7.23.x/multiuser/keycloak/che-multiuser-keycloak-user-remover/src/main/java/org/eclipse/che/multiuser/keycloak/server/KeycloakUserRemover.java#L89. But maybe it's not an issue if hosted Che doesn't supports this feature at all.